javaparser-core/src/main/java/com/github/javaparser/ast/CompilationUnit.java

```diff
@@ -72,6 +73,13 @@ public CompilationUnit(int beginLine, int beginColumn, int endLine, int endColum
         setTypes(types);
     }
 
+	public CompilationUnit(Position begin, Position end, PackageDeclaration pakage, List<ImportDeclaration> imports, List<TypeDeclaration> types) {
```

Comment:
maybe package_ (in the code I have seen an underscore used after the identifier when it would correspond to a keyword)